controller/instance_manager_controller.go

```diff
@@ -535,6 +531,21 @@ func (imc *InstanceManagerController) syncLogSettingsToInstanceManagerPod(im *lo
 	}
 	defer client.Close()
 
+	if types.IsDataEngineV1(im.Spec.DataEngine) {
+		settingName := types.SettingNameLogLevel
+		setting, err := imc.ds.GetSettingWithAutoFillingRO(settingName)
+		if err != nil {
+			return err
+		}
+
+		err = client.LogSetLevel(longhorn.DataEngineTypeV1, "", setting.Value)
+		if err != nil {
+			return errors.Wrapf(err, "failed to set instance-manager log level from %v", settingName)
+		}
+
+		return nil
+	}
```

Comment:
Why can this setting only be applied to v1 data engine? v2 data engine should be applied as well.

Reply:
Ah, thank you.

Reply:
Remedied and re-pushed.